src/main/java/core/basesyntax/service/FruitTransaction.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public FruitTransaction setOperation(Operation operation) {
+        this.operation = operation;
+        return this;
```

Comment:
The use of chained setters (returning `this` in setters) is not typical for simple POJO classes and might lead to confusion. It is more commonly used in Builder patterns.